policyengine_us/parameters/gov/states/nd/tax/income/rates/single.yaml

```diff
@@ -40,5 +49,7 @@ metadata:
     href: https://www.tax.nd.gov/sites/www/files/documents/forms/form-nd-1-2022.pdf
```

Comment:
Could you change all labels from ND to North Dakota while you are in here

Reply:
Got it.